packages-ui/roosterjs-react/lib/rooster/component/Rooster.tsx

```diff
@@ -13,13 +13,13 @@ import { useTheme } from '@fluentui/react/lib/Theme';
  */
 export default function Rooster(props: RoosterProps) {
     const editorDiv = React.useRef<HTMLDivElement>(null);
-    const editor = React.useRef<IEditor>(null);
+    const editor = React.useRef<IEditor | null>(null);
```

Comment:
Is this required change? I think useRef will add null to the allowed the value type automatically

Reply:
It is, otherwise the ref is read-only
<img width="619" alt="image" src="https://user-images.githubusercontent.com/44042957/191814316-7b45debb-3757-48dc-b081-3964eb3c389f.png">

Reply:
ok